bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3089,9 +3094,13 @@ namespace bftEngine
 
 
 				uint32_t actualReplyLength = 0;
-
-				int error = userRequestsHandler->execute(clientId, req.isReadOnly(), req.requestLength(), req.requestBuf(),
-															maxReplyMessageSize - sizeof(ClientReplyMsgHeader), replyBuffer, actualReplyLength);
+				// Yulia TBD: sequenceNum should come from corresponding State
+				// Metadata object.
+				int error = userRequestsHandler->execute(
```

Comment:
Instead of 0. Pass lastExecutedSeqNum + 1

Reply:
Done.